ios/objc/testApp/objc/TestApp/TestApp/utils/ImagePredictor.mm

```diff
@@ -0,0 +1,69 @@
+#import "ImagePredictor.h"
+#import <torch/script.h>
+#import <PytorchObjC/PytorchObjC.h>
+#import <vector>
+#import "UIImage+Utils.h"
+
+#define IMG_W 224
```

Comment:
Nit: use static consts instead, like:
`static const NSUInteger kImageWidth = 224;`
Or even use a const of type `CGSize` and define both.